### ISSUE TITLE
I can not reproduce the AP=0.295 with your advice. Do I miss something important?

### ISSUE THREAD

**Issue by KeyKy** [endernewton/tf-faster-rcnn#12]
I read your code and try to reproduce the AP in caffe-version py-faster-rcnn. In caffe-version py-faster-rcnn, I set RPN_MIN_SIZE=0 to keep the small proposals. My parameter of anchor is ratios=[0.5,1,2] scales: [4, 8, 16, 32]. My Learning rate setting is:
base_lr:0.001
lr_policy: "multistep"
stepvalue: 350000
stepvalue: 600000
stepvalue: 900000
gamma:0.1
weight_decay: 0.0005
momentum: 0.9
iter_size:2

I set the ASPECT_GROUPING: True and train with multi-scales.
I train with proposal_layer and test with proposal_layer(not proposal_top_layer).

I stop training at 1000K, because the valid ap stop improving. Although this setting can give me a improvement in small objects but the all AP=0.261(1000K). I train the vgg16 on trainval-minival and test on minival. Do I miss something?

**Comment by endernewton** [comment]
You may need to exclude the GT boxes in proposals?

**Comment by KeyKy** [comment]
What do you mean exclude the ground truth boxes in proposal? Could you explain it more detail? Thanks.

**Comment by endernewton** [comment]
Haha I think that's why I wrote that report. You can check that out and see what is your current implementation differences.

**Comment by KeyKy** [comment]
@endernewton Ho, I found you use crop_and_resize instead of roi_pooling_layer. Do you compare TF+roi_pooling_layer with TF+crop_and_resize?

**Comment by endernewton** [comment]
It's slightly better. Again, please check the report, I have written down all the differences that seem to give potential performance boost.
